### PR TITLE
Amend shebang line in sandbox scripts for python2

### PR DESCRIPTION
**Pull request by Echelon9**
Resolves Issue #794 

- [x] Is it mergable?
- [x] Did it pass the tests?
- [x] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage.
- [x] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [x] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [x] Is it documented in the ChangeLog?
- [x] Was a spellchecker run on the source code and documentation after
  changes were made?

### CODE REVIEW

**Comment by Echelon9** [comment]
Ready for review

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+2015-02-21  Rhys Kidd  <rhyskidd@gmail.com>
+
+   * sandbox/{calc-best-assembly,calc-error-profile,normalize-by-align,
+   read_aligner,slice-reads-by-coverage}.py: reference /usr/bin/env python2
+   in the shebang line.
```
Comment (mr-c) [inline ChangeLog]: 'in the #! line.' please, thanks!

**Comment by Echelon9** [comment]
PR amended per feedback from @mr-c

**Comment by mr-c** [comment]
Thanks!

**Pull request merged**